src/api.js

```diff
@@ -1,13 +1,27 @@
-import axios from 'axios'
+import axios from "axios";
 
-const ncNewsApi = axios.create({baseURL: "https://be-nc-news-project-j8hl.onrender.com/api"})
+const ncNewsApi = axios.create({
+  baseURL: "https://be-nc-news-project-j8hl.onrender.com/api",
+});
 
 export const getArticles = () => {
-    return ncNewsApi.get("/articles")
+  return ncNewsApi
+    .get("/articles")
     .then((articles) => {
-        return articles.data
+      return articles.data;
     })
     .catch((error) => {
-        console.log(error)
+      console.log(error);
+    });
+};
+
+export const getArticle = (article_id) => {
+  return ncNewsApi
+    .get(`/articles/${article_id}`)
+    .then((requestedArticle) => {
+      return requestedArticle;
     })
-}
\ No newline at end of file
+    .catch((error) => {
+      console.log(error);
+    });
```

Comment:
We shouldnt be dealing with the error here. We should let this function fail so we can have a .catch(...) on the component side and deal with that error there in some way.